### ISSUE TITLE
fix flutter failures (before 0.1.89)

### ISSUE THREAD

**Issue by pq** [dart-lang/linter#1570]
The changes to `prefer_null_aware_operators` are flagging a few new lints.  We'll need to fix these before we land in the SDK (so as not to block integration).

```
⏩ RUNNING: cd .; bin/flutter analyze --dartdocs --flutter-repo --dart-sdk /b/s/w/ir/cache/builder/sdk/out/ReleaseX64/dart-sdk
Analyzing 3 directories...                                      

   info • Prefer using null aware operators • packages/flutter/lib/src/gestures/arena.dart:269:27 • prefer_null_aware_operators
   info • Prefer using null aware operators • packages/flutter/lib/src/painting/gradient.dart:609:7 • prefer_null_aware_operators
   info • Prefer using null aware operators • packages/flutter/test/material/dropdown_test.dart:66:20 • prefer_null_aware_operators
   info • Prefer using null aware operators • packages/flutter_tools/lib/src/ios/simulators.dart:521:16 • prefer_null_aware_operators
```

https://logs.chromium.org/logs/dart/buildbucket/cr-buildbucket.appspot.com/8912972001568965456/+/steps/analyze_flutter/0/stdout

/cc @a14n

**Comment by a14n** [comment]
Sorry I made a mistake in flutter/flutter#32711 (I commented out the wrong lint) that should be fixed by flutter/flutter#33026

**Comment by pq** [comment]
No problem.  Thanks for the quick fix!

**Comment by pq** [comment]
![image](https://user-images.githubusercontent.com/67586/58040444-6790f580-7aea-11e9-81e5-a7ab66ffd4c6.png)
🍻
